custom_typings/dom.d.ts

```diff
@@ -0,0 +1,11 @@
+declare interface Window {
+  customElements: CustomElementRegistry;
```

Comment:
but either ways we could start adding them to the library and get better/more accurate declarations as Edge picks them up.